site/docs/components/menu/usage.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Using the component
+
+### When to use
+
+- When you need to display a list of actions or options.
+
+### When not to use
+
+- Inside a form. Instead, use [`ComboBox`](../combo-box) or [`Dropdown`]("../dropdown).
+
+## Content
+
+- Always use sentence case for menu item labels.
+- Keep menu item labels short and concise.
+
+## Import
+
+To import `Menu` and related components from the lab Salt package, use:
+
+```js
+import {
+  Menu,
+  MenuItem,
+  MenuTrigger,
+  MenuPanel,
+  MenuGroup,
+} from "@salt-ds/lab";
+```
+
+## Props
+
+### `Menu`
+
+<PropsTable packageName="lab" componentName="Menu" />
```

Comment:
nit: an probably remove these if there are no props

Reply:
I was conflicted about this, I feel like a component should always have a props table even if it is empty but it's worth being consistent.

Reply:
Will it ever have props?

Reply:
I just checked the preview and it looks like there are props for this component?